x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/index.tsx

```diff
@@ -58,23 +58,22 @@ export const HostDetailsFlyout = memo(() => {
 
   useEffect(() => {
     if (error !== undefined) {
-      notifications.toasts.danger({
+      toasts.addDanger({
         title: (
           <FormattedMessage
             id="xpack.securitySolution.endpoint.host.details.errorTitle"
             defaultMessage="Could not find host"
           />
         ),
-        body: (
+        text: (
           <FormattedMessage
             id="xpack.securitySolution.endpoint.host.details.errorBody"
             defaultMessage="Please exit the flyout and select an available host."
           />
         ),
-        toastLifeTimeMs: 10000,
```

Comment:
oh nice! so does the newer toast notification already have some default toast lifetime built in?

Reply:
yep